CMakeLists.txt

```
@@ -152,12 +152,12 @@ if (ENABLE_TESTING AND UNIX AND NOT APPLE)
         setup_target_for_coverage_gcovr_xml(
                 NAME coverage
                 EXECUTABLE ctest
-                DEPENDENCIES qa_buffer qa_data_sink qa_dynamic_port qa_dynamic_node qa_hier_node qa_filter qa_settings qa_tags qa_scheduler qa_thread_pool qa_thread_affinity
+                DEPENDENCIES qa_buffer qa_data_sink qa_DynamicPort qa_DynamicBlock qa_HierBlock qa_filter qa_settings qa_tags qa_Scheduler qa_thread_pool qa_thread_affinity
```

The scheduler test was not renamed to uppercase and this breaks the coverage CI build. Should be either changed back here or the test renamed.

```suggestion
                DEPENDENCIES qa_buffer qa_data_sink qa_DynamicPort qa_DynamicBlock qa_HierBlock qa_filter qa_settings qa_tags qa_scheduler qa_thread_pool qa_thread_affinity
```

---

done